### DISCUSSION TITLE
Edit same bot's message

### DISCUSSION THREAD

**Discussion by lafabo** [python-telegram-bot/python-telegram-bot#2593] [Q&A] [answered]
Hi! I'm writing simple bot: after /start it would once send a wellcome message and than once a minute bot will edit this first wellcome message and replacing it's text with a current time, for example.

How it could be done?

**Comment by Bibo-Joshi** [comment, marked as answer]
You can do this e.g. with PTBs built-in `JobQueue`. Resources: [wiki page](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions-–-JobQueue), [example](https://github.com/python-telegram-bot/python-telegram-bot/tree/master/examples#timerbotpy), [docs](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.jobqueue.html)

**Reply by lafabo** [reply]
thanks!
